force-app/main/lwc-utils/lwc/tableServiceUtils/tableServiceUtils.js

```diff
@@ -41,7 +41,7 @@ const applyLinks = (flatData) => {
   for (let row of dataClone) {
     Object.keys(row).forEach(key => {
       if (key.endsWith('Id')) {
-        row[key] = '/'+row[key];
+        row[key.replace('Id','Link')] = '/'+row[key];
```

Comment:
@shliachtx is this just about making the property logical? (i.e. AccountId being a link doesn't make sense but AccountLink does)

Reply:
that, and leaving the Id field unchanged (for updates)